PluginsAndFeatures/azure-toolkit-for-intellij/build.gradle

```diff
@@ -140,14 +140,6 @@ configure(mainProjects) {
         jvmArgs '-Xmx2048m'
     }
 
-    instrumentCode {
-        // javac2 is required to compile .form files from IntelliJ IDEA UI Designer
```

Comment:
Hm... Don't we still need this to compile Java `.forms` code? MS side is built on forms.

Reply:
```
A problem was found with the configuration of task ':instrumentCode' (type 'IntelliJInstrumentCodeTask').
> File '....\.gradle\caches\modules-2\files-2.1\com.jetbrains.intellij.idea\ideaIC\211.4961.30-EAP-SNAPSHOT\cc7e56bf686f8783c0471f36d85083748d44b387\ideaIC-211.4961.30-EAP-SNAPSHOT\lib\javac2.jar' specified for property 'javac2' does not exist.
```

Reply:
So not sure :(